examples/child-design-system/README.md

```diff
@@ -10,94 +10,97 @@ This is the reasoning behind creating the CMS design system family which consist
-However the content of the page can be changed at the child design system level. For example, to change the landing page of the child design system doc site you would create a markdown page named `index.md` then provide your own content for that particular page. You can see a list of core design system doc site pages here. https://github.com/CMSgov/design-system/tree/master/packages/design-system-docs/src/pages
+### Configuring tests
 
-**Note:** The page name must be identocal and live in the same location in order for it to be overridden at a child design system level.
+Unit tests need a `setupTests.js` file to be defined in your design system source folder. This file is run before each test file via Jest's `setupFilesAfterEnv` config option](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array), and is used to configure your tests. CMSDS scripts supports 2 options for component testing by default, either [Enzyme](https://enzymejs.github.io/enzyme/) or [React Testing Library](https://testing-library.com/).
 
-If you are writing a content only page then you should be creating a Markdown .MD page but if you are writing documentation for a component, pattern or utility you should follow the [format outlined for writing component documentation](https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md).
+## Writing documentation
 
-### Publishing your child design system doc site
+By default the pages that exist in the core design system will be a part of the child design system documentation site. However, content can be overriden at the child design system level. For example, to change the "Installation" page of the child design system doc site under "Getting Started", create a markdown page named `installation.md` under `docs/src/pages/startup` containing your custom content for that particular page. You can reference all the core design system doc site pages [here](https://github.com/CMSgov/design-system/tree/master/packages/design-system-docs/src/pages).
+
+**Note:** The page name must have the same name and same location as the original page for it to be overridden at a child design system level.
 
-We suggest using GitHub pages `gh-pages` to publish the doc site build files.
+If you are writing a content only page then you should be creating a Markdown (`.md`) file. However if you are are writing documentation for a component, pattern or utility you should follow the [KSS format outlined for writing component documentation](https://github.com/CMSgov/design-system/blob/master/guides/WRITING-DOCUMENTATION.md).
 
-### Modifying the design
+We have also provided example documentation for an overriden `Button` component, a new `Card` component, and a new `border-style` utility for reference.
+
+### Publishing your child design system doc site
 
-- (Color variables)[https://github.com/CMSgov/design-system/blob/master/packages/design-system/src/styles/settings/variables/_color.scss]
```

Comment:
I felt like this information was already covered on the theming page, and didnt necessarily flow with the rest of the content on the page. However, I think it would make sense to maybe make a new section about customizing the DS that links the theming info, as well as how to customize the components

Reply:
Gotcha, yeah that makes sense. I def don't want us to have multiple versions of documentation.